avm/res/document-db/database-account/tests/e2e/role/dependencies.bicep

```diff
@@ -0,0 +1,77 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Managed Identity to create.')
+param keyVaultName string
+
+@description('Required. The name of the App Service to create.')
+param appName string
+
+@description('Optional. Type of site to deploy.')
+@allowed([
+  'functionapp' // function app windows os
+  'functionapp,linux' // function app linux os
+  'functionapp,workflowapp' // logic app workflow
+  'functionapp,workflowapp,linux' // logic app docker container
+  'app,linux' // linux web app
+  'app' // normal web app
+])
+param appKind string = 'app,linux'
+
+@description('Required. The name of the App Service Plan to create.')
+param appServicePlanName string
+
+@description('Optional. The kind of the App Service Plan.')
+param appServicePlanKind string = ''
+
+@description('Optional. If Linux app service plan , otherwise.')
+param appServicePlanReserved bool = true
+
+@description('Optional. Managed service identity.')
+param managedIdentity bool = !empty(keyVaultName)
```

Comment:
Your test always needs a managed identity. Can we remove this parameter and always emit an identityPrincipalId without a condition?